jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/DecoderHelper.java

```diff
@@ -0,0 +1,107 @@
+package us.dot.its.jpo.ode.coder;
```

Comment:
This class is BSM specific. Should it be called BsmDecoderHelper?
OR
Since everything is about BSM, perhaps we need to put all these refactored calsses in coder.bsm package.